packages/material-ui/src/Typography/typography.spec.tsx

```diff
@@ -14,6 +14,9 @@ const TypographyTest = () => {
       <Typography align="inherit" color="textSecondary" />
       <Typography align="inherit" color="error" />
       // $ExpectError
+      <Typography align="inherit" color="warning" />
```

Comment:
Can you move those down so that "new tests" = "last tests"? Right now an old test no longer expects an error while a new one does.

Reply:
@eps1lon fixed. there is just one error in circleci now, nopt sure what it means though